### ISSUE TITLE
Find a new theme for the SOBotics blog

### ISSUE THREAD

**Issue by superplane39** [SOCVFinder/SOCVFinder.github.io#28]
Let's face it. The current theme for the [SOBotics blog](http://blog.sobotics.org/) is... not pretty. In fact, it's pretty ugly and basic. It's okay for the very start of the blog; it's not getting high traffic, so it doesn't need to look *too* professional.

But still... we should strive for something that looks better.

So, please suggest some themes you like down below. Feel free to thumbs-up designs you like. This is just to gather some options - not actually changing anything quite yet, so feel free to suggest even the most ridiculous stuff :P

**Comment by Filnor** [comment]
What is the backend again? Wordpress, Jekyll or something else?

**Comment by superplane39** [comment]
Powered by Jekyll.

**Comment by Bhargav-Rao** [comment]
The one feature which I definitely need is the RSS feeds. Any template with RSS feeds is fine for me.

**Comment by double-fault** [comment]
This seems to be quite good: https://github.com/aron-bordin/neo-hpstr-jekyll-theme. Seems to have an RSS feed as well.

**Comment by iBug** [comment]
Jekyll theme? Grab a random one and modify it! We don't lack HTML and CSS developers `:)`

**Comment by ArtOfCode-** [comment]
> Grab ~~a random one~~...

Grab Charcoal's theme, change the logo and colours, use your own links and boom

**Comment by iBug** [comment]
@ArtOfCode- that could be ridiculously useful! ~~I could even write a gemspec and wrap up a gem for easier distribution (if needed)~~

**Comment by Filnor** [comment]
@iBug Yes please! Most of us are not very good at Ruby (if they even have used it) but installing a Gem should be easy enough.

**Comment by iBug** [comment]
@Filnor Seriously, are you agreeing with Art? His words *looks* a bit like joking to me. But yeah, I'm serious on the words that I can wrap Charcoal's theme up for our use.

Or are you responding to my previous comment (grab a random one and modify)?

**Comment by Filnor** [comment]
@iBug I missed the irony there (but hey, I just got up :sweat_smile:). But in seriousness, why take the hard way when we can have the easy one?

**Comment by iBug** [comment]
@Filnor yeah really. But I'm not good at aesthetics so maybe you guys find one you like and I can help with tweaking.

**Comment by artofcode** [comment]
> @artofcode that could be ridiculously useful! I could even write a gemspec and wrap up a gem for easier distribution (if needed)

@ArtOfCode- , not me... again :-)

**Comment by ArtOfCode-** [comment]
Heh. Sorry @artofcode :D
